micromanager_gui/_core_widgets/_presets_widget.py

```diff
@@ -0,0 +1,65 @@
+from typing import Optional, Tuple
+
+from qtpy.QtWidgets import QComboBox, QHBoxLayout, QWidget
+from superqt.utils import signals_blocked
+
+from .._core import get_core_singleton
+
+
+class PresetsWidget(QWidget):
+    """Create a QCombobox Widget for a specified group presets"""
+
+    def __init__(
+        self,
+        group: str,
+        parent: Optional[QWidget] = None,
+    ) -> None:
+
+        super().__init__(parent)
+
+        self._mmc = get_core_singleton()
+
+        self._group = group
+
+        if self._group not in self._mmc.getAvailableConfigGroups():
+            raise ValueError(f"{self._group} group does not exist.")
+
+        self._presets = list(self._mmc.getAvailableConfigs(self._group))
+
+        if not self._presets:
+            raise ValueError(f"{self._group} group does not have presets.")
+
+        self._combo = QComboBox()
+        with signals_blocked(self._combo):
+            self._combo.addItems(self._presets)
```

Comment:
```suggestion
        self._combo.addItems(self._presets)
```

in this case, you don't need it because you haven't yet connected the combo box to core (that happens on line 39)